Mesh_3/include/CGAL/Mesh_3/Refine_facets_3.h

```diff
@@ -356,6 +358,19 @@ class Refine_facets_3_base
     return sstr.str();
   }
 
+  const int dimension(const Intersection& intersection) const
```

Comment:
The return type `const int` will certainly trigger a warning with one compiler or another. I suggest:

```suggestion
  int dimension(const Intersection& intersection) const
```